guides/source/working_with_javascript_in_rails.md

```diff
@@ -56,14 +56,16 @@ technique. You rarely have to write this code yourself. The rest of this guide
 will show you how Rails can help you write websites in this way, but it's
 all built on top of this fairly simple technique.
 
-Unobtrusive JavaScript
+Unobtrusive JavaScript (UJS)
 ----------------------
 
 Rails uses a technique called "Unobtrusive JavaScript" to handle attaching
 JavaScript to the DOM. This is generally considered to be a best-practice
 within the frontend community, but you may occasionally read tutorials that
 demonstrate other ways.
 
+NOTE: To enable unobtrusive JavaScript on a page, use `Rails.start()`. We do not encourage use of any other UJS function, as they may change over time.
```

Comment:
This section explains Unobtrusive JavaScript as a general concept, separate from any particular implementation; a note about the public API of `rails-ujs` seems out of place here.

I'm sceptical about the value of adding this note, since the existing guide doesn't mention `window.Rails` at all. It's hard to imagine how reading this guide could give someone the impression that `Rails.ajax` was public API.

@jhackett1 were you asking for that note because it would have you saved you time personally, or to discourage others from writing articles that demonstrate using `Rails.ajax` in application code (like the one you linked to)?

Reply:
I was a little hesitant about this note because I also think its a bit out of place. Maybe we can put it in the Rails-ujs readme instead, although it would be great if we could bake this into the library somehow. I don't know how to denote public and private API in JavaScript. 🤔

Reply:
@javan is there a preferred way to denote public/private API for rails-ujs?